docs/CHANGELOG.rst

```diff
@@ -10,6 +10,11 @@ This project adheres to `Semantic Versioning <http://semver.org/>`_.
 Unreleased
 ==========
 
+Added
+-----
+- Add ``resdk`` Python 3 package to ``resolwebio/utils`` Docker image
```

Comment:
IMO, It is sufficient to just write `Add resdk Python package to resolwebio/utils Docker image`. The package might also be used for other purposes than just clustering... But i don't have a strong opinion about this.